test/e2e/cli.go

```diff
@@ -15,10 +15,16 @@
 package e2e
```

Comment:
From an organization perspective, I am wondering if it would make sense to keep interactive test helper functions in a separate file? Maybe keep `cli.go` as where to define things that will be needed by all testing helper commands? WDYT?